km_api/know_me/serializers/emergency_contact_serializers.py

```diff
@@ -0,0 +1,32 @@
+"""Serializers for the ``EmergencyContact`` model.
+"""
+
+from rest_framework import serializers
+
+from know_me import models
+
+
+class EmergencyContactListSerializer(serializers.HyperlinkedModelSerializer):
+    """
+    Serializer for ``EmergencyContact`` instances.
+    """
+    url = serializers.HyperlinkedIdentityField(
+        view_name='know-me:emergency-contact-detail')
+
+    class Meta:
+        fields = ('id', 'url', 'name', 'relation', 'phone_number',
+                  'alt_phone_number', 'email')
```

Comment:
Another style correction here: Multi-line tuples are written with the opening and closing parentheses on separate lines.
```python
fields = (
    'id', 'url', 'name', 'etc'
)
```